weather/Weather.Viz.py

```diff
@@ -0,0 +1,33 @@
+# Import necessary libraries 
+import pandas as pd 
+import requests 
+
+# Create and define the class
+class Weather_Viz: 
+    def __init__(self, API_url, API_token):
+        # Initiate the API_url and API_token
+        self.API_url = API_url
+        self.API_token = API_token
+        # Fetch the data and store it in the DF propetry
+        self.data_frame = self.get_data()
+
+    def get_data(self): 
+        # Define the headers for API request
+        headers = {
+            'Authorization' : f'Bearer {self.API_token}',
```

Comment:
So for the weather API we are not using a bearer token, it's just an API key and the parameter name is just "Key" not "Authorization" and the Key goes inside "params" instead of "Headers".